src/app/georeferencer/qgsimagewarper.cpp

```diff
@@ -240,7 +240,8 @@ int QgsImageWarper::warpFile( const QString &input,
   progressDialog->raise();
   progressDialog->activateWindow();
 
-  eErr = oOperation.ChunkAndWarpMulti( 0, 0, destPixels, destLines );
+  eErr = oOperation.ChunkAndWarpImage( 0, 0, destPixels, destLines );
+//  eErr = oOperation.ChunkAndWarpMulti(0, 0, destPixels, destLines);
```

Comment:
```suggestion
  eErr = oOperation.ChunkAndWarpImage( 0, 0, destPixels, destLines );
```

Reply:
Strangely I cannot apply the suggestion in github. Though leaving the comment might not be this bad, I think we should maybe mention why it doesn't work instead, for an eventual followup. Does that sound like a better option?

Reply:
The normal way in the QGIS codebase is to rip off unused code. I think it's better to stick to the standards and open an issue.

Reply:
Very well, text removed. Thanks